src/types.ts

```diff
@@ -1 +1,4 @@
-export type ThemeColorOption = "white" | "gray";
+type ThemeColorOption = "white" | "gray" | "blue";
+type TitleColorOption = "blue";
```

Comment:
It's a little bit strange to define a new option type like this with only one option, especially when this is not actually the actual default color that we use on most components. Could you add `black` as an option here so that it's clearer what the normal set of options are?